exercises/practice/bob/.meta/config.json

```diff
@@ -1 +1,19 @@
-{"authors": ["axtens"], "files": {"solution": ["src/bob.cob"], "test": ["test.sh", "test.ps1"], "example": [".meta/proof.ci.cob"]}, "blurb": "Bob is a lackadaisical teenager. In conversation, his responses are very limited.", "source": "Inspired by the 'Deaf Grandma' exercise in Chris Pine's Learn to Program tutorial.", "source_url": "http://pine.fm/LearnToProgram/?Chapter=06"}
\ No newline at end of file
+{
+    "authors": [
+        "axtens"
+    ],
+    "files": {
+        "solution": [
+            "src/bob.cob"
+        ],
+        "test": [
+            "tst/bob/bob.cut"
+        ],
+        "example": [
+            ".meta/proof.ci.cob"
+        ]
```

Comment:
I would recommend adding the `test.sh` and `test.ps1` files to the (optional) `invalidator` key: https://exercism.org/docs/building/tracks/practice-exercises#h-file-meta-config-json That way, any time those files change, the student is prompted to update the exercise.

Reply:
I'll open another PR to add those.